test/ut/sdk/test_model_speedup.py

```diff
@@ -177,7 +177,6 @@ def channel_prune(model):
     pruner.compress()
     pruner.export_model(model_path=MODEL_FILE, mask_path=MASK_FILE)
 
-@unittest.skipIf(torch.__version__ >= '1.6.0', 'not supported')
```

Comment:
Could you check if the test cases work with PyTorch 1.6+ in following scripts?
  - test_graph_utils.py
  - test_compression_utils.py
  - test_pruners.py
  - test_dependecy_aware.py

Reply:
pytorch 1.6+ turned on for 
```
test_compression_utils.py
test_pruners.py
test_dependecy_aware.py
```

for `test_graph_utils.py`,  there are some expected file are generated by Yuge, they may need to be upgraded for pytorch 1.6+

@ultmaster ,  would you please help upgrade test_graph_utils.py for pytroch1.6+ ?